test/Core/Eval/Base.hs

```diff
@@ -10,21 +10,30 @@ import Juvix.Compiler.Core.Info qualified as Info
 import Juvix.Compiler.Core.Info.NoDisplayInfo
 import Juvix.Compiler.Core.Language
 import Juvix.Compiler.Core.Pretty
+import Juvix.Compiler.Core.Transformation
 import Juvix.Compiler.Core.Translation.FromSource
 import System.IO.Extra (withTempDir)
 import Text.Megaparsec.Pos qualified as M
 
-coreEvalAssertion :: FilePath -> FilePath -> (String -> IO ()) -> Assertion
-coreEvalAssertion mainFile expectedFile step = do
+coreEvalAssertion ::
+  FilePath ->
+  FilePath ->
+  [TransformationId] ->
+  (InfoTable -> Assertion) ->
+  (String -> IO ()) ->
+  Assertion
+coreEvalAssertion mainFile expectedFile trans testTrans step = do
   step "Parse"
   r <- parseFile mainFile
   case r of
     Left err -> assertFailure (show (pretty err))
     Right (_, Nothing) -> do
-      step "Compare expected and actual program output"
+      step "Compare expected an actual program output"
```

Comment:
Why the change? "and" was fine, now it's not grammatical

Reply:
typo. Will fix it